### DISCUSSION TITLE
LAB#A-E 配分詢問

### DISCUSSION THREAD

**Discussion by Ponponri** [bol-edu/HLS-SOC-Discussions#155] [Q&A] [answered]
請問新後的配分是 LAB#A-E 做兩個各8分，加起來16分嗎? 目前表上 LAB#A-E 的總分只有8分。

![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/51708838/a85f7a6a-95e0-4d95-877f-240706dc321a)

**Comment by Lanxas1018** [comment, marked as answer]
同學好
更新後的配分只剩LabD可以選擇做或不做，其他的LabABCE已經取消
